<ui version="4.0" >
 <class>SchemaConfigurationDialog</class>
 <widget class="QDialog" name="SchemaConfigurationDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure Schema</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="iterationBox" >
      <property name="title" >
       <string>Iterations</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_333" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QGroupBox" name="dataBox" >
      <property name="title" >
       <string>Parameters</string>
      </property>
      <property name="flat" >
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth" >
          <number>1</number>
         </property>
         <property name="childrenCollapsible" >
          <bool>false</bool>
         </property>
         <widget class="QTreeView" name="treeView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy" >
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers" >
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="tabKeyNavigation" >
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="rootIsDecorated" >
           <bool>true</bool>
          </property>
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTreeView" name="dataView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>10</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers" >
           <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
          </property>
          <property name="tabKeyNavigation" >
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="textElideMode" >
           <enum>Qt::ElideMiddle</enum>
          </property>
          <property name="horizontalScrollMode" >
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="uniformRowHeights" >
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SchemaConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SchemaConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>collapsed(QModelIndex)</signal>
   <receiver>dataView</receiver>
   <slot>collapse(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>107</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel" >
     <x>302</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeView</sender>
   <signal>expanded(QModelIndex)</signal>
   <receiver>dataView</receiver>
   <slot>expand(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>107</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel" >
     <x>302</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataView</sender>
   <signal>collapsed(QModelIndex)</signal>
   <receiver>treeView</receiver>
   <slot>collapse(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>425</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataView</sender>
   <signal>expanded(QModelIndex)</signal>
   <receiver>treeView</receiver>
   <slot>expand(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>425</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
